### PR TITLE
fix in tf.keras.applications.EfficientNetXX

### PR DESCRIPTION
**Pull request by joonb14**
in tensorflow/tensorflow/python/keras/applications/efficientnet.py
at line 316,317 there is

x = layers.Rescaling(1. / 255.)(x)
x = layers.Normalization(axis=bn_axis)(x)

We normally put normalized data as input, and other networks(like mobilenet and resnet) other than this don't have such layers.
I think these two layers should be removed.

### CODE REVIEW

**Comment by bhack** [comment]
https://github.com/tensorflow/tensorflow/blob/d8df26c4311abc332cc04d463d8154ab60c19014/tensorflow/python/keras/applications/efficientnet.py#L160-L165

I suppose this requires to re-save trained model right?

**Comment by qlzh727** [comment]
Please see for more details about why we doing this.

https://github.com/tensorflow/tensorflow/pull/47808#pullrequestreview-612848161

**Comment by joonb14** [comment]
> Please see for more details about why we doing this.
> 
> [#47808 (review)](https://github.com/tensorflow/tensorflow/pull/47808#pullrequestreview-612848161)

Thanks for the reply! However, I am concerned that users who used backbone, such as mobilenet v2 and resnet, which did not have preprocessing layers, may not notice this change just like me. To make sure everyone who utilize tf.keras.applications.EfficientNet, MobileNetV3 realize this change, I think it would be better to provide a parameter that gives the user a choice whether or not to include a preprocessing layer. 
image
Thank you!

**Comment by qlzh727** [comment]
Ack. Will send a fix soon.

**Comment by qlzh727** [comment]
@joonb14. just checked with API owner @fchollet about this. Adding a new flag for whether to include preprocessing will make the API surface misaligned with other models. Currently the only proper usage of all the models are:

```
xxx = application_module
model = xxx.Model()(xxx.process_input(data))
```

For you concern about other models that doesn't include preprocessing, it will very likely to get a low performance if user didn't invoke process_input(), which is exactly why we want to include the processing in the model.
